string param_name
int16 value
---
bool result